recipes/btyacc/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+import functools
+import os
+
+from conans import CMake, ConanFile, tools
+
+required_conan_version = ">=1.43.0"
+
+
+class BtyaccConan(ConanFile):
+    name = "btyacc"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ChrisDodd/btyacc"
+    description = "Backtracking yacc"
+    topics = ("yacc", "parser")
+    license = "???"
+    settings = ("os", "arch", "compiler", "build_type")
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+    generators = ("cmake",)
+    exports_sources = ("CMakeLists.txt",)
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        root = self._source_subfolder
+        get_args = self.conan_data["sources"][self.version]
+        tools.get(**get_args, destination=root, strip_root=True)
```

Comment:
Less is more?

```suggestion
        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
```

Reply:
No can do. PEP 8 requires 79 line length.

Reply:
the suggestion doesn't provide any functional improvements, only code style changes